src/parameters.toml

```diff
@@ -494,41 +494,23 @@ type = "float"
 
 # Microphysics - Modal aerosol model
 
-[mam3_r_dry_coarse]
-alias = "r_dry_coarse"
-value = 1e-6
-type = "float"
-description = "Mean dry radius for the coarse modal distribution of MAM3"
-
-[mam3_r_dry_accum]
-alias = "r_dry_accum"
-value = 0.243e-6
-type = "float"
-description = "Mean dry radius for the accumulation modal distribution of MAM3"
-
-[mam3_r_dry_ait]
-alias = "r_dry_ait"
-value = 0.05e-6
-type = "float"
-description = "Mean dry radius for the aitken modal distribution of MAM3"
-
```

Comment:
Can we split this PR into breaking and non-breaking changes?

Reply:
I don't think its a breaking change. They have never been merged on main

Reply:
Sorry, I added the parameters in the initial commit of this PR, but Anna rightly pointed out that they don't make sense to include. This shouldn't be a breaking change.